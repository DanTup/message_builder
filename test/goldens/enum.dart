class SomeEnum {
  static const anotherValue = const SomeEnum._(2);
  static const someValue = const SomeEnum._(1);
  final int _value;
  const SomeEnum._(this._value);
  factory SomeEnum.fromJson(int value) {
    const values = const {2: SomeEnum.anotherValue, 1: SomeEnum.someValue};
    return values[value];
  }
  int toJson() => _value;
}
