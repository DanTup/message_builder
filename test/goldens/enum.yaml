SomeEnum:
  wireType: int
  enumValues:
    someValue: 1
    anotherValue: 2
